uint16[] id_list

string[] drive_mode

uint8[] led                    
uint8[] registered_instruction 
float64[] realtime_tick_us          
uint8[] moving                 
uint8[] moving_status          

float64[] return_delay_time_us
float64[] homing_offset_deg
float64[] moving_threshold_deg_s
bool[]  restore_configuration 
uint8[] shutdown               
uint8[] status_return_level    
float64[] bus_watchbdog_ms          